routers/api-router.js

```diff
@@ -9,8 +12,13 @@ apiRouter
   })
   .get("/topics", getTopics)
   .get("/articles/:article_id", getArticleById)
+  .patch("/articles/:article_id", patchArticle)
   .all("/*", (req, res) => {
     res.status(404).send({ msg: "Path not found" });
   });
 
+apiRouter.use((err, req, res, next) => {
+  console.log(err, "<-- err in apiRouter");
```

Comment:
Always a good idea to remove console.logs as soon as you don't need them anymore to keep your code nice and clean

Reply:
Done :)